model/src/main/proto/profile.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Top level proto used to store last used id and all profiles
+message Profiles {
+  // Represents the last used unique ID for adding a profile
+  int32 last_used_id = 1;
+
+  // Mapping from unique ID to profile
+  map<int32, Profile> profiles = 2;
+}
+
+// Structure for a single profile
+message Profile {
+  // Unique ID given to each profile
+  int32 id = 1;
```

Comment:
Shouldn't this be a ``ProfileId``? Otherwise I'm not sure how calling code will pass in a ``ProfileId`` to select a profile to log into.